### DISCUSSION TITLE
classes/class_worldboundaryshape3d

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#291] [User-contributed notes]
# classes/class_worldboundaryshape3d

Inherits: Shape3D< Resource< RefCounted< Object A 3D world boundary (half-space) shape used for physics collision. Description: A 3D world boundary shape, intended for use in physics. WorldBoundary...

https://docs.godotengine.org/en/stable/classes/class_worldboundaryshape3d.html

**Comment by BlackShift** [comment]
As a note for jolt users, the boundary is not infinite, there is a setting under limits that restricts the size.

**Reply by Calinou** [reply]
Pull request opened:

- https://github.com/godotengine/godot/pull/101108
